### PR TITLE
Jhancock/multi sig/fixes

### PR DESCRIPTION
**Pull request by dhancockFigure**
fixed an overflow on a small screen, added disabled text color, and added a ui for when there are no multisig accounts to recover

### CODE REVIEW

File: lib/l10n/app_en.arb [hunk 1]
```diff
@@ -635,5 +635,7 @@
   "devConfirmProposal": "Confirm Proposal",
   "devProposalComplete": "PROPOSAL COMPLETE",
   "devEnableMultiSig": "Enable Multi Sig",
-  "sendAssetListLoadingAssets": "Loading assets"
+  "sendAssetListLoadingAssets": "Loading assets",
+  "pleaseNote": "Please note",
```
Comment (jakegraves-figure) [inline lib/l10n/app_en.arb:639]: It's been the standard to prefix the name of the screen to the name of the String, like so:
`"pleaseNote": "Please note",` -> `"multiSigRecoverPleaseNote": "Please note",`

File: lib/screens/home/accounts/multi_account_item.dart [hunk 2]
```diff
@@ -162,13 +162,6 @@ class _MultiAccountItemState extends State<MultiAccountItem> {
                   Navigator.of(context).pop(MenuOperation.select);
                 },
               ),
-            PwListDivider(),
-            PwGreyButton(
-              enabled: false,
-              text: '',
-              // ignore: no-empty-block
-              onPressed: () {},
-            ),
```
Comment (jakegraves-figure) [inline lib/screens/home/accounts/multi_account_item.dart:171]: This wasn't doing anything, true, but it was there to match the design. Does this change the way the menu looks? If so, this needs to be reverted. Or at least changed so it matches the way it used to look.

**Review by jakegraves-figure — CHANGES_REQUESTED** [review]
Something small, and something that may affect the design.

File: lib/l10n/app_en.arb (hunk 1 above)
Comment (dhancockFigure) [reply lib/l10n/app_en.arb:639]: ok will make that change

File: lib/screens/home/accounts/multi_account_item.dart (hunk 2 above)
Comment (dhancockFigure) [reply lib/screens/home/accounts/multi_account_item.dart:171]: on smaller screens it caused the menu to be partially scrolled. This obscured part of the available actions to show blank space

File: lib/l10n/app_en.arb [hunk 3]
```diff
@@ -635,5 +635,7 @@
   "devConfirmProposal": "Confirm Proposal",
   "devProposalComplete": "PROPOSAL COMPLETE",
   "devEnableMultiSig": "Enable Multi Sig",
-  "sendAssetListLoadingAssets": "Loading assets"
+  "sendAssetListLoadingAssets": "Loading assets",
+  "multiSigRecoverPleaseNote": "Please note",
```
Comment (royd) [inline lib/l10n/app_en.arb:639]: Might be nice to place these next to the other `multiSig` prefixed items

File: lib/screens/add_account_flow.dart [hunk 4]
```diff
@@ -281,6 +279,7 @@ class AddAccountFlowState extends FlowBaseState<AddAccountFlow>
     );
 
     if (link != null) {
+      String _multiSigInvalidLink = Strings.of(context).multiSigInvalidLink;
```
Comment (royd) [inline lib/screens/add_account_flow.dart:282]: Could remove the underscore to keep with convention and also make this `final`, or remove the local variable altogether.

File: lib/screens/multi_sig/multi_sig_recover_screen.dart [hunk 5]
```diff
@@ -133,29 +154,31 @@ class _MultiSigRecoverScreenState extends State<MultiSigRecoverScreen> {
...
+            recoverableRemoteAccounts[remoteAccount.remoteId] =
+                _MultiSigAccountData(
+              account: remoteAccount,
+              linkedAccount: basicAccount,
+            );
+          }
         }
       }
-    }
+    } else {}
```
Comment (royd) [inline lib/screens/multi_sig/multi_sig_recover_screen.dart:181]: Could remove empty `else`

**Review by royd — CHANGES_REQUESTED** [review]

**Review by royd — APPROVED** [review]

**Pull request merged**